public/js/machine.js

```diff
@@ -1 +1,28 @@
-$('#machine-table').tablesort();
+// $('#machine-table').tablesort();
```

Comment:
Was this `tablesort` function not doing anything? If so, it's fine to comment it out. But, if it was doing something, we should probably talk about whether comment it out is a good idea.

Reply:
Yeah {#  <script src="/js/machine.js"></script>#} is commented out in machine.njk so I don't think this function was ever called.